From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sun, 10 Aug 2014 13:07:23 +0200
Subject: changing gridvis.conf with additional comment

---
 etc/gridvis.conf | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/etc/gridvis.conf b/etc/gridvis.conf
index c70530e..e456fe5 100644
--- a/etc/gridvis.conf
+++ b/etc/gridvis.conf
@@ -1,6 +1,8 @@
 # ${HOME} will be replaced by user home directory according to platform
-default_userdir="${HOME}/.${APPNAME}/6.0"
-default_mac_userdir="${HOME}/Library/Application Support/${APPNAME}/6.0"
+#default_userdir="${HOME}/.${APPNAME}/6.0"
+# We set a global project directory in /var/lib/GridVisProjects,
+# also storing the license information there.
+default_userdir="/var/lib/GridVisProjects/.${APPNAME}/6.0"
 
 # options used by the launcher by default, can be overridden by explicit
 # command line switches
@@ -9,10 +11,10 @@ default_options="--branding gridvis -J-Xms64m -J-Xmx1024m -J-Dvisualvm.display.n
 # for development purposes you may wish to append: -J-Dnetbeans.logger.console=true -J-ea
 
 # default location of JDK/JRE, can be overridden by using --jdkhome <dir> switch
-#jdkhome="jre"
+# If the required JRE version can't be satisfied by Debian set jdkhome to
+# 'jre', else uncomment the definination.
+jdkhome="jre"
 
 # clusters' paths separated by path.separator (semicolon on Windows, colon on Unices)
 #extra_clusters=
-jdkhome="jre"
 
-        
\ No newline at end of file
